misc/python/materialize/cloudtest/k8s/testdrive.py

```diff
@@ -50,6 +52,7 @@ def run(
         no_reset: bool = False,
         seed: Optional[int] = None,
         caller: Optional[Traceback] = None,
+        default_timeout: str = "300s",
```

Comment:
Is this actually correct to specify the timeout using this parameter? Why is it called `default_timeout` and not `timeout`? Is it overridden somewhere?

Reply:
yes, you can override it inside a .td file, 
```
$ set-sql-timeout duration=60s
```

`default_timeout` seemed like an appropriate name when this functionality was added to testdrive, as this timeout applies to   various different operations. Maybe `common_timeout` would have been more descriptive. Renaming it to just `timeout` now would be several-hundred-line PR right now I think.